src/features/pastelDB/pastelDBLib.ts

```diff
@@ -215,11 +215,67 @@ export function validateDataFromDB(
             return false
           }
           break
+        case 'blocksubsidy':
```

Comment:
This function is becoming too big. Please split it and test all split functions separately